uwblueprint/paramedics-web
# Update role names

[Notion ticket](https://www.notion.so/uwblueprintexecs/update-user-access-levels-1488ff68ce2a419a85c82d2f690b21d2)

## Changes
- Updated `role` and `user` models to establish associations between the two
- Added `role` queries to `user` resolver
- Created migration to change column in `users` table from `accessLevel` to `roleId`, which now uses the `roles` table
- Merge together with [frontend change](https://github.com/uwblueprint/paramedics-react/pull/45)
## Screenshots

## Testing
1. Run migration and verify that `users` table has `accessLevel` column removed and is replaced with `roleId` column
2. `roleId` column should have correct mappings - 1 is Commander, 2 is Supervisor (no mapping for dispatch)
3. Verify that `roles` query works in GraphQL
4. Verify frontend tests on branch `update-user-roles` in `paramedics-react` from respective frontend PR

## Checklist
>[Code review doc for reference](https://www.notion.so/uwblueprintexecs/Code-Review-a21fc85b00394f488e92d9d605f6b2bc)

before opening PR

filling out PR

after opening PR
